shoes-core/spec/shoes/builtin_methods_spec.rb

```diff
@@ -106,4 +78,49 @@
       end
     end
   end
+
+  %w(alert confirm).each do |type|
+    describe '##{alert}' do
+     let(:dialog) { double('dialog') }
+     let(:message) { double('message') }
```

Comment:
sure, I sometimes prefer doubles since the also can catch changes a normal String wouldn't: so if you pass in `'message'` and someone introduces `message.lowercase` in the implementation, the double will catch that (reminding the implementor to adept the test accordingly), but as said a matter of preference